raiden-ts/src/path/epics.ts

```diff
@@ -81,64 +83,67 @@ export const pathFindServiceEpic = (
                 }).pipe(
                   timeout(httpTimeout),
                   mergeMap(async response => {
+                    const text = await response.text();
                     if (!response.ok)
                       throw new Error(
-                        `PFS: paths request: code=${
-                          response.status
-                        } => body=${await response.text()}`,
+                        `PFS: paths request: code=${response.status} => body="${text}"`,
                       );
-                    const decoded = PathResults.decode(await response.json());
-                    if (isLeft(decoded)) throw ThrowReporter.report(decoded);
-                    return decoded.right;
+                    return decode(PathResults, losslessParse(text));
                   }),
-                  map(({ result }) => ({
-                    routes: result
-                      .slice()
-                      .sort(({ estimated_fee: a }, { estimated_fee: b }) => a - b)
-                      .map(({ path }) => ({ route: path })),
-                  })),
+                  map(
+                    (results: PathResults): Paths => ({
```

Comment:
Would be nice for developers if they can still receive the routes already sorted by fees

Reply:
The routes are generaly sorted by fees. I removed our custom sorting here because the PFS knows a couple of heuristics to better sort this end result, so I decided to pass it through. The cleanup below is mostly to limit to what we can do (e.g. single output node) and/or revalidate in case PFS sent routes which we locally know can't be used anymore, but overall we respect the PFS hints and sorting.